Microsoft.Azure.Cosmos/src/Resource/CosmosExceptions/CosmosOperationCanceledException.cs

```diff
@@ -87,12 +88,12 @@ public override Exception GetBaseException()
         /// <inheritdoc/>
         public override string ToString()
         {
-            return $"{this.originalException} {Environment.NewLine}CosmosDiagnostics: {this.Diagnostics}";
+            return this.Message;
         }
 
-        private Lazy<string> CreateLazyMessage()
+        private Lazy<string> CreateLazyMessage(CancellationToken token)
         {
-            return new Lazy<string>(() => $"{this.originalException.Message} {Environment.NewLine}CosmosDiagnostics: {this.Diagnostics}");
+            return new Lazy<string>(() => $"{this.originalException.Message} Cancellation Token has expired: {token.IsCancellationRequested}. Learn more at: https://aka.ms/cosmosdb-tsg-request-timeout {Environment.NewLine}CosmosDiagnostics: {this.Diagnostics}");
```

Comment:
Before, the newline was used to separate the message from the diagnostics. Since we are adding now a new piece of information (the token state and link), should we maintain the same pattern with the newline after the original Message?

```suggestion
            return new Lazy<string>(() => $"{this.originalException.Message}{Environment.NewLine}Cancellation Token has expired: {token.IsCancellationRequested}. Learn more at: https://aka.ms/cosmosdb-tsg-request-timeout{Environment.NewLine}CosmosDiagnostics: {this.Diagnostics}");
```